VespaGAS/Managers/OilMixCalculator.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+let MLInLiter: Float = 1000
+
+typealias Oil = Volume
+typealias Gasoline = Volume
+
+struct OilMixCalculator {
+    static func oilFor(gasoline: Gasoline, oilMix: OilMix) -> Oil {
```

Comment:
Cool! Yeah having domain objects will make it easier to see things like this. Domain-driven design is really powerful and you'll have noticed we used it significantly in OC.

You could add this behaviour to `Oil` or `Gasoline` but that would mean altering the underlying `Volume` type right? Are there any other domain objects this behaviour could make sense on?